moler/config/loggers.py

```diff
@@ -243,6 +284,10 @@ def configure_moler_main_logger():
                                   formatter=MultilineWithDirectionFormatter(fmt=debug_log_format,
                                                                             datefmt=date_format))
 
+        global moler_logo
```

Comment:
Why you add global declaration here? You don't modify `moler_logo` from inside of this function - you just read it. So, global line is unnecessary.